Header header
float64 speed
int32 rawSpeed
